agent/src/main/scala/za/co/absa/atum/agent/com/dummy/Person.scala

```diff
@@ -0,0 +1,17 @@
+
+package za.co.absa.atum.agent.com.dummy
+
+case class Person(
```

Comment:
I think the above is enough to have some dummy code. But no problem if you keep this one too.